packages/trader/src/Modules/Trading/Components/Form/TradeParams/barrier.jsx

```diff
@@ -41,6 +40,7 @@ const Barrier = ({
     const format = value => {
         const float_value = parseFloat(value);
         let final_value;
+
```

Comment:
nit-pick) we can undo the changes to this file now )

Reply:
Done, thank you 👍🏻